agent/fw_wordpress.c

```diff
@@ -285,7 +285,13 @@ static char* nr_wordpress_plugin_from_function(zend_function* func TSRMLS_DC) {
     nrl_verbosedebug(NRL_FRAMEWORK,
```

Comment:
`nrl_verbosedebug()` appears to be based on `vsnprintf()` under the hood and I am concerned (depending on the libc in use) that passing `NRP_PHP((char *)nr_stack_get_top(&NRPRG(wordpress_tags))` without checking it for NULL might be passing NULL to "%s" which could be a problem.  Are we certain this can't happen at this point in the code?

Reply:
NRP_PHP checks for NULL and passes an empty string if so